content/02.introduction.md

```diff
@@ -1,12 +1,26 @@
 ## Introduction
 
-1. Talk about knowledge graphs
-  1. Define what they are
-  2. Provide a diagram to orient the reader
-2. Mention how these graphs are constructed 
-3. Mention some applications these graphs can be used for
-	1. Biomedical setting
-	2. Social Media 
-	3. Question and answering systems 
-	4. more if it comes to mind
-4. Conclude section by mentioning that this review dives deeper into approaches used for construction and applying these graphs to biomedical applications
+Knowledge graphs are a practical resource for a multitude of real world applications.
+This resource utilizes graph theory to represent knowledge in the form of a graph.
```

Comment:
If I understand "This resource" correctly, it says "Knowledge graphs utilize graph theory to represent knowledge in the form of a graph."

Not sure you need this sentence, but I do think the sentence afterwards defining a knowledge graph is helpful. Plus it is probably good to add some references to knowledge graphs here and give just a little bit of info about them. What does and what doesn't qualify?